dev-python/argon2_cffi/argon2_cffi-18.3.0.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{2_7,3_{4,5,6,7}} pypy pypy3 )
+inherit distutils-r1
+
+DESCRIPTION="CFFI bindings to the Argon2 password hashing library"
+HOMEPAGE="https://github.com/hynek/argon2_cffi"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc test"
+
+CDEPEND="
+	~app-crypt/argon2-20171227:=
+	dev-python/six[${PYTHON_USEDEP}]
+	$(python_gen_cond_dep '>=dev-python/cffi-1.0.0[${PYTHON_USEDEP}]' 'python*')
+	$(python_gen_cond_dep 'dev-python/enum34[${PYTHON_USEDEP}]' 'python2_7' 'pypy')
```

Comment:
`virtual/python-enum34`? ;-)

Reply:
Didn't know these virtuals exist. Thanks for the hint!